draft-ietf-scitt-architecture.md

```diff
@@ -126,63 +126,47 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 
 # Introduction
 
-This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
-It achieves this goal by enforcing the following complementary security guarantees:
-
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+This document describes the scalable, flexible, and decentralized SCITT architecture.
+Its goal is to enhance auditability and accountability across supply chains.
+
+In supply chains, there are producers and consumers of items.
+Consumers may like to have information about the items that they consume, such as an item's provenance.
+SCITT provides a way for consumers to obtain this information in a way that is "transparent", that is, producers cannot lie about the information without it being detected.
+SCITT achieves this by having producers (also called Issuers) publish information in a Transparency Service, where consumers (also called Verifiers) can check the information.
+
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts they release, for example: "Issuer name", "Artifact type", "Artifact version".
+1. Issuers sign the Statements, producing Signed Statements.
+1. Issuers submit the Signed Statements to Transparency Services.
+1. Transparency Services take the submitted Signed Statements and insert them into an Append-only Log, producing Transparent Statements.
+They may add their own metadata such as: "Transparency Service name", "Artifact registration time", "registration policy used".
+This process can be thought of as Transparency Services counter-signing Issuers' Signed Statements.
+Transparency Services have a Registration Policy controlling what can be submitted.
+They may also have an authorization policy controlling who can access their service.
+1. Verifiers consume the Transparent Statements, verifying them in order to be convinced of the Statement about the Artifact.
+1. Auditors verify the correctness and consistency of the Transparent Statements.
+
+An entity can take on multiple roles, for example, an entity can act both as an Issuer and as a Verifier.
```

Comment:
I wonder if this could be changed to a bulleted list:
```suggestion
An entity can take on multiple roles, for example:

- A Verifier may make new Statements for what they verified
- An Issuer can make multiple Statements about the same Artifact
- An Issuer can make amended Statements about the same Artifact as their view changes over time
- Multiple Issuers can make different, even conflicting Statements, about the same Artifact where another verifier may choose which Issuers they trust
- Multiple Issuers can make the same Statement about a single Artifact, affirming multiple Issuers agree
```

Reply:
This is good.

Reply:
A few nits:

- Imho the bullets are not examples of "an entity taking on multiple roles at the same time", only the first one is. The others are examples of what a given role can do.
- And even the first bullet is imprecise: the Verifier cannot make Statements, because by design the Issuer does that.  Entity != role.
- The second and third bullet are kind the same / the third is a specification of the second.

Iterated proposal:

```
An entity can take on multiple roles.
For example, an entity can first act as a Verifier, and later act as an Issuer by making a Statement about the verification outcome.

Issuers are free to do any Statements. In particular:

- An Issuer can make multiple Statements about the same Artifact. For example, an Issuer can make amended Statements about the same Artifact as their view changes over time
- Multiple Issuers can make different, even conflicting Statements, about the same Artifact. Verifiers can choose which Issuers they trust
- Multiple Issuers can make the same Statement about a single Artifact, affirming multiple Issuers agree
```

While I like this, I feel like it is getting too much for the introduction again. Can we somehow fit these as clarifying examples into section 2 or  5?

Reply:
The above looks like a cleaner update. 
I'd remove the following line, as the bullets say the same, and the wording of "free to do any" would need updates.

Reply:
Pulled out to #152